### ISSUE TITLE
Coach - Non-Premium and Non-Verified - Click on CLubs Signed athetle or College Commited - Chnage message 

### ISSUE THREAD

**Issue by stevekamps** [technicalbirdVayuz/volleyball-unite-usa-issue-tracker#185]
Remove the alert and 2 bullet points  below:

Alert
- To avail premium benefits you need to verify your profile first.

- Request an assocaition or already verified coach to verify your profile.

Change to the following:

To search for signed/committed Athletes... Go Premium Now!!

Then when they click on the Premium tab list all features that they get for Premium.  Then when they click on "Go Premium" have a message come up that says ......

"You need to be a "Verified Coach" before you can purchase a Premium membership.  To receive your "Verification Badge" (place green icon here) and get your Coaching Profile "Verified" by any Organization on the Volleyball Unite Platform... Simply "Search" for an Organization and send a request to become "Verified".  

Once verified you can upgrade to a Premium Membership.

**Comment by technicalbirdVayuz** [comment]
Its conflicting with https://github.com/technicalbirdVayuz/volleyball-unite-usa-issue-tracker/issues/123#issuecomment-915256610

Please check both & let me know what needs to be done

**Comment by stevekamps** [comment]
Change to the following for a coach (club or College)

To search for signed/committed Athletes... Go Premium Now!!

Then when they click on the Premium tab list all features that they get for Premium. Then when they click on "Go Premium" have a message come up that says ......

"You need to be a "Verified Coach" before you can purchase a Premium membership. To receive your "Verification Badge" (place green icon here) and get your Coaching Profile "Verified" by any Organization on the Volleyball Unite Platform... Simply "Search" for an Organization and send a request to become "Verified".

Once verified you can upgrade to a Premium Membership.


I would like an athlete to be able to go Premium without having to be Verified.  But please still have the verification process.  
Once Verified and Premium I would like to have the blue and green checkmarks showing on all player profile areas/badges.  Also make the green check mark look just like the blue checkmark

**Comment by stevekamps** [comment]
I will delete Issue #123

**Comment by stevekamps** [comment]
Still not fixed.  
Change to the following for a coach (club or College)

To search for signed/committed Athletes... Go Premium Now!!

Then when they click on the Premium tab list all features that they get for Premium. Then when they click on "Go Premium" have a message come up that says ......

"You need to be a "Verified Coach" before you can purchase a Premium membership. To receive your "Verification Badge" (place green icon here) and get your Coaching Profile "Verified" by any Organization on the Volleyball Unite Platform... Simply "Search" for an Organization and send a request to become "Verified".

Once verified you can upgrade to a Premium Membership.

I would like an athlete to be able to go Premium without having to be Verified. But please still have the verification process.
Once Verified and Premium I would like to have the blue and green checkmarks showing on all player profile areas/badges. Also make the green check mark look just like the blue checkmark

**Comment by technicalbirdVayuz** [comment]
Putting it on Hold, because of no action taken

**Comment by stevekamps** [comment]
Please remove from hold and make this change.  We don't want an athlete to have to wait for a coach to verify or track one down if they are excited to be on the platform and want to buy a premium.

**Comment by technicalbirdVayuz** [comment]
> > @technicalbirdVayuz had a discussion remove the athlete check to go premium only if verified. @stevekamps confirm here please
> 
> Please share it on the sheet, will share the time for this iteration first before starting work @rgnarok @stevekamps

@rgnarok Please share this on the sheet. We will share the timeline before starting the work

**Comment by technicalbirdVayuz** [comment]
> > @technicalbirdVayuz had a discussion remove the athlete check to go premium only if verified. @stevekamps confirm here please
> 
> Please share it on the sheet, will share the time for this iteration first before starting work @rgnarok @stevekamps

@rgnarok Please share this on the sheet. We will share the timeline before starting the work, once done will remove the hold

**Comment by rgnarok** [comment]
https://docs.google.com/spreadsheets/d/1t9_46wYkk0_tLDxHOXeAwS3DFnnrpRyQSbBroGlO2iQ/edit#gid=410465843
@technicalbirdVayuz

**Comment by technicalbirdVayuz** [comment]
We have made the required changes in Web. Please review.

Also, if there are any changes required in the notification then let us know

**Comment by technicalbirdVayuz** [comment]
Once closure given on web, we will add the same flow in app
